### ISSUE TITLE
[Question] Maximum Random Seed

### ISSUE THREAD

**Issue by nejt** [chancejs/chancejs#365]
What is the maximum seeded value for Chance? 
Is it capped at JavaScript's largest integer value (2^53-1)?

I have been seeding it with Ethereum addresses.  I took a look at the code and noticed lines 48-63 about how the seed is built.  Thus I am wondering if I could overflow/max out the seed value...

**Comment by victorquinn** [comment]
Yes, if a number, it would be capped at JavaScript's largest integer value.

That said, the seed can also take the form of a string so you could seed it with the hex value of the address as a string and that should work for you.

For example:

```js
var chance = new Chance("0x281055Afc982d96fAB65b3a49cAc8b878184Cb16");
// now use chance!
```

**Comment by nejt** [comment]
Thank you for the quick response!
But it looks like the code turns a string into a number?
```
// create a numeric hash for each argument, add to seedling     
var hash = 0;
for (var k = 0; k < arguments[i].length; k++) {
  hash = arguments[i].charCodeAt(k) + (hash << 6) + (hash << 16) - hash;
}
seedling += hash;
```
